readme.md

```diff
@@ -242,6 +242,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [rbpf](https://github.com/qmonnet/rbpf) - Written in Rust. Interpreter for Linux, macOS and Windows, and JIT-compiler for x86_64 under Linux.
 - [PREVAIL](https://github.com/vbpf/ebpf-verifier) - A user space verifier for eBPF [using an abstract interpretation layer](https://elazarg.github.io/pldi19main-final.pdf), with support for loops.
 - [oster](https://github.com/grantseltzer/oster) - Written in Go. A tool for tracing execution of Go programs by attaching eBPF to uprobes.
+- [wachy](https://rubrikinc.github.io/wachy/) - A tracing profiler that aims to make eBPF uprobe-based debugging easier to use. This is done by connecting it back to the source code and providing a UI for interactive drilldown analysis.
```

Comment:
> This is done by connecting it back to the source code

Not clear to me what this means, I don't believe this means instrumenting the code, right? You mean displaying the code in the UI, a bit like when annotating with `perf`?